code/addons/interactions/src/components/MethodCall.tsx

```diff
@@ -341,7 +341,7 @@ export const ElementNode = ({
 };
 
 export const DateNode = ({ value }: { value: string | Date }) => {
-  const [date, time, ms] = (value instanceof Date ? value.toISOString() : value).split(/[T.Z]/);
+  const [date, time, ms] = new Date(value).toISOString().split(/[T.Z]/);
```

Comment:
This change assumes value can always be converted to a valid Date. However, if value is an invalid date string, new Date(value) will result in an invalid Date object (Invalid Date).

It can lead to runtime errors if value is not a valid date string. Consider adding error handling for invalid dates.

Suggestion: To handle potential invalid date values, add error handling and validate the date conversion.

Something like this:

```
export const DateNode = ({ value }: { value: string | Date }) => {
  let dateStr, time, ms;

  try {
    const dateObj = new Date(value);
    if (isNaN(dateObj.getTime())) {
      throw new Error('Invalid date');
    }
    [dateStr, time, ms] = dateObj.toISOString().split(/[T.Z]/);
  } catch (error) {
    console.error('Invalid date value provided to DateNode:', value);
    [dateStr, time, ms] = ['Invalid date', '', ''];
  }

  const colors = useThemeColors();
  return (
    <span style={{ whiteSpace: 'nowrap', color: colors.date }}>
      {dateStr} {time} {ms}
    </span>
  );
}
```

Reply:
My previous version was a bit simpler and I could revert to it:

`const [date, time, ms] = (value instanceof Date ? value.toISOString() : value).split(/[T.Z]/); `

would this work?

Reply:
Thx @jadehamel , updated PR with catch for invalid date.

Reply:
This is a good suggestion. I think we can even make do without the try/catch by omitting the `getTime` call.